ydb/library/yql/udfs/common/knn/knn-distance.h

```diff
@@ -12,10 +12,57 @@
 #include <util/generic/buffer.h>
 #include <util/stream/format.h>
 
+#include <bit>
+
 using namespace NYql;
 using namespace NYql::NUdf;
 
-static std::optional<float> KnnManhattanDistance(const TStringRef& str1, const TStringRef& str2) {
+inline void BitVectorHandleShort(ui64 byteLen, const ui64* v1, const ui64* v2, auto&& op) {
+    Y_ASSERT(0 < byteLen);
+    Y_ASSERT(byteLen < sizeof(ui64));
```

Comment:
1. Please, double check that ALL the Y_ASSERT are not dependent on user input.
2. Replace all Y_ASSERT to Y_DEBUG_ASSERT. It's common practice for UDF modules.

Reply:
1. Done
2. I didn't find Y_DEBUG_ASSERT in the codebase. Also Y_ASSERT already nop when NDEBUG defined

Reply:
2. Oh, it's all right. Y_ASSERT is already in DEBUG only.